src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,34 +2,40 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+    public static final int INT = 11;
+    public static final int INT1 = -1;
+    public static final int twelve = 12;
+    private final int[] numbers = new int[twelve];
 
-    public int total = -1;
+    private int total = INT1;
 
-    public void countIn(int in) {
+    protected void countIn(int in) {
         if (!isFull())
-            NUMBERS[++total] = in;
+            getNumbers()[++total] = in;
```

Comment:
This is an exaggeration. Using private fields internally usually do not require encapsulation.